Moments/NewMomentsPage.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  NewMomentsPage.swift
+//  Moments
+//
+//  Created by user on 01/11/17.
+//  Copyright © 2017 user. All rights reserved.
+//
+
+import UIKit
+
+class NewMomentsPage: UITableViewController , UITextFieldDelegate {
+    
+    @IBOutlet weak var momentName: UITextField!
+    
+    @IBOutlet weak var momentDescription: UITextField!
+    
+    @IBOutlet weak var datePicker: UIDatePicker!
+    @IBOutlet weak var toolBar: UIToolbar!
+    
+    @IBOutlet weak var chooseDate: UILabel!
+    @IBOutlet weak var color: UILabel!
+    
+    @IBOutlet weak var datePickerBottomConst: NSLayoutConstraint!
+    @IBOutlet weak var viewForPicker: UIView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        toolBar.setValue(true, forKey: "hidesShadow")
+        viewForPicker.isHidden = true
+        datePickerBottomConst.constant = -267
+        
+        color.layer.cornerRadius = color.frame.size.width / 2
+        
+        //hides the navigation bar hairline
+        self.navigationController?.navigationBar.setValue(true, forKey: "hidesShadow")
+        
+        momentName.returnKeyType = UIReturnKeyType.next
```

Comment:
you can configure this in the storyboard itself no need to add programatically

Reply:
i changed it to storyboard